### DISCUSSION TITLE
why i recieved an this error??

### DISCUSSION THREAD

**Discussion by NSH531** [dotnet/runtime#73753] [Q&A] [answered]
`1>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(4632,5): warning MSB3026: Could not copy "obj\Debug\OU.exe" to "bin\Debug\OU.exe". Beginning retry 3 in 1000ms. The process cannot access the file 'bin\Debug\OU.exe' because it is being used by another process. `

**Comment by NSH531** [comment]
I attached a screenshot with this error 

![image](https://user-images.githubusercontent.com/5903577/184090302-fd170b48-9082-45e4-9007-8d4274a5ef2d.png)

**Comment by En3Tho** [comment, marked as answer]
You received it because that exe is currently running on some other process. Perhaps you started it in console and forgot to close the app

**Comment by NSH531** [comment]
this code is thrown this error:
```

var p=   Process.Start("U:\\Downloads\\WinFormsApp1\\OU\\powershell.exe",
"U:\\Downloads\\WinFormsApp1\\OU\\cmd.ps1"+" > "+
"U:\\Downloads\\WinFormsApp1\\OU\\cmd."+DateTime.Now.ToShortDateString()+".out");
            p.Start();
            p.BeginOutputReadLine();
            var outh = p.StandardOutput;
            Console.WriteLine(outh.ReadToEnd());


```


‫בתאריך יום ה׳, 11 באוג׳ 2022 ב-11:42 מאת ‪Igor Bagdamyan‬‏ <‪
***@***.***‬‏>:‬

> You received it because that exe is currently running on some other
> process. Perhaps you started it in console and forgot to close the app
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/runtime/discussions/73753#discussioncomment-3372868>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABNBJWPV3NIL34CGG577B2TVYS4JDANCNFSM56HGFQWA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
<https://netanel.ml>

**Comment by NSH531** [comment]
שבוע טוב :-)

אני רוצה לעדכן אותך שלצערי אני לא יכול בשלב זה להמשיך להעסיק את מנהל השיווק
שלי

שמו אור באבד.

הוא עושה עבודה נהדרת!

אני ממליץ עליו בחום וחושב שהוא יכול להיות נכס עבורך.

אור חי בטורונטו קנדה ועובד בשלט רחוק

הוא בוגר תואר מדעי המחשב מאוניברסיטת תל אביב ויש לו ניסיון של 15 שנה בניהול
קמפיינים בישראל ובעולם.

 ניתן לקבוע פגישה בקישור הבא:

https://calendly.com/tigertest-hungary/meet-or-babad
<https://api.mailbrother.com/mail/click/NnN5QXpjcGFwaExhcm9vcEZVQkZmWW9wUVQ5bmZBTjdaQnFUUnd1a2FVcmY5NXQ5ejdsRnVxdW5YaE05TDNPU3ZYRng4L3E3bDlpUitlU21sWUxEQnhTM3AzSGF1Ui9KbG1CT0t2UzV5MnYxeERUeHNkcWhTajR5dXRqQk42YitYZmZRSzgyY2VnU0FjN2xWYzJPUkF3PT0=>

אני אשמח שתתאמו פגישה.

אני רוצה שתכירו!!!

המייל של אור:

***@***.***



Sent with MailBrother
<https://chrome.google.com/webstore/detail/mailbrother-html-inserter/obngoldljmnnpggbekneikaohbeflbee>
